### ISSUE TITLE
Why did you increase the batch size for training?

### ISSUE THREAD

**Issue by SebastianJanampa** [Peterande/D-FINE#215]
Hello,
Thanks for such awesome work. I read the paper, and you said you used a batch size of 32 for 72 epochs. What's the reason for using 32 instead of a batch size 16 (as in previous DETRs models)? Also, using a batch_size of 16, you should only need to train for 36 epochs (since it keeps the same number of iterations) which would show the DFINE has faster convergence than RT-DETR.
Regards,
Sebastian

**Comment by Peterande** [comment]
Training convergence is independent of iterations. When the batch size is halved, the learning rate should also be halved, and other parameters will be affected as well. Generally speaking, using the largest total batch size that can be accommodated will accelerate convergence.
